commit f054a7df29b99932cbe50c1216d545029b2e769c (refs/changes/44/4544/3)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2018-07-30T13:20:03+01:00 (1 year, 2 months ago)
    
    MANTA-3795 nginx build broken on recent platform images
    Reviewed By: Robert Mustacchi <rm@joyent.com>
    Approved By: Robert Mustacchi <rm@joyent.com>

diff --git a/src/http/modules/mpu/deps/json-nvlist.c b/src/http/modules/mpu/deps/json-nvlist.c
index 10847b02..1b4ffc1a 100644
--- a/src/http/modules/mpu/deps/json-nvlist.c
+++ b/src/http/modules/mpu/deps/json-nvlist.c
@@ -10,7 +10,7 @@
  */
 
 /*
- * Copyright 2015 Joyent, Inc.
+ * Copyright 2018 Joyent, Inc.
  */
 
 #include <stdio.h>
@@ -19,7 +19,6 @@
 #include <strings.h>
 #include <errno.h>
 #include <libnvpair.h>
-#include <sys/ccompile.h>
 
 #include "json-nvlist.h"
 
@@ -337,7 +336,7 @@ hdlr_bareword(state_t *s)
 /* ARGSUSED */
 static int
 collect_number(state_t *s, boolean_t *isint, int64_t *result,
-    double *fresult __GNU_UNUSED)
+    double *fresult)
 {
 	boolean_t neg = B_FALSE;
 	int64_t t;
